tests/dashbio_demos/app_molecule2d.py

```diff
@@ -0,0 +1,230 @@
+import os
+import json
+
+from dash.dependencies import Input, Output, State
+import dash_html_components as html
+import dash_core_components as dcc
+
+import pubchempy as pcp
+
+from dash_bio_utils.chem_structure_reader import read_structure
+import dash_bio
+
+# running directly with Python
+if __name__ == '__main__':
+    from utils.app_standalone import run_standalone_app
+
+# running with gunicorn (on servers)
+elif 'DASH_PATH_ROUTING' in os.environ:
+    from tests.dashbio_demos.utils.app_standalone import run_standalone_app
+
+DATAPATH = os.path.join(".", "tests", "dashbio_demos", "sample_data", "mol2d_")
+
+
+def description():
+    return 'Two-dimensional visualizations of molecular structure.'
```

Comment:
```suggestion
    return 'Two-dimensional visualization of molecular structures.'
```

Reply:
Fixed in a23cc81!

Reply:
Since a23cc81 doesn't contain any other change and has a non-compliant commit message, it would have been better (in this case) to commit via the interface the lazy way...